### PR TITLE
#245 - updated multiformats, @ipld/dag-cbor, and interface-blockstore dependencies

### PR DESCRIPTION
**Pull request by thehenrytsai**

### CODE REVIEW

File: src/utils/cid.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ const codecs = {
  * @throws {Error} encoding fails
  * @throws {Error} if hasher is not supported
  */
-export async function computeCid(payload: any, codecCode = cbor.code, multihashCode = sha256.code): Promise<string> {
+export async function computeCid(payload: any, codecCode: number = cbor.code, multihashCode: number = sha256.code): Promise<string> {
```
Comment (diehuxx) [inline src/utils/cid.ts:32]: Nit: this line is on the long side. do you mind linebreaking it somewhere?

File: src/store/message-store-level.ts [hunk 2]
```diff
@@ -73,7 +73,7 @@ export class MessageStoreLevel implements MessageStore {
       return undefined;
     }
 
-    const decodedBlock = await abortOr<block.Block>(options?.signal, block.decode({ bytes, codec: cbor, hasher: sha256 }));
```
Comment (diehuxx) [inline src/store/message-store-level.ts:76]: I think usages of `block` are all removed, but we're still importing. mind removing?

While on the subject, what do you think of[ adding a plugin](https://www.npmjs.com/package/eslint-plugin-unused-imports) to the linter to catch unused imports? I can file an issue and add a PR.

**Review by diehuxx — APPROVED** [review]
A couple non-blocking comments. Otherwise LGTM

File: src/utils/cid.ts (hunk 1 above)
Comment (thehenrytsai) [reply src/utils/cid.ts]: Done. Note that we do have a linter rule on max 150 chars.

Comment (diehuxx) [reply src/utils/cid.ts]: ah didn't realize. i'm used to closer to 80-100 max chars

File: src/store/message-store-level.ts (hunk 2 above)
Comment (thehenrytsai) [reply src/store/message-store-level.ts:76]: I think it does get used, `block.decode()` in same line. Also we already have a linter rule `@typescript-eslint/no-unused-vars` includes this check, I get nagged all the time when imports are not used.

**Review by diehuxx — APPROVED** [review]

File: src/utils/cid.ts (hunk 1 above)
Comment (thehenrytsai) [reply src/utils/cid.ts]: 80 to 100?!?! That was the standard length limit back when I was in college in 2000 when the screen size was 15" and resolution was 1024x768..... we now have 32"+ monitors with crazy resolutions!! = ]

**Pull request merged**